### PR TITLE
Bump version to 0.0.288

### PR DESCRIPTION
**Pull request by MichaReiser**

### CODE REVIEW

**Comment by MichaReiser** [comment]
Sorry. I had to rename the branch to avoid a name conflict with the tag

**Comment by MichaReiser** [comment]
Current dependencies on/for this PR:
* main
  * **PR #7271** <a href="https://app.graphite.dev/github/pr/astral-sh/ruff/7271" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/astral-sh/ruff/7271?utm_source=stack-comment).

File: BREAKING_CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Breaking Changes
 
+## 0.0.288
+
+### Remove support for emoji identifiers ([#7212](https://github.com/astral-sh/ruff/pull/7212))
+
+Previously, Ruff supported the non-standard compliant emoji identifiers `📦.show()`.
```
Comment (zanieb) [inline BREAKING_CHANGES.md:7]: Maybe?
```suggestion
Previously, Ruff supported the non-standard compliant emoji identifiers e.g. `📦 = 1`.
```

File: BREAKING_CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+We decided to remove this non-standard language extension, and Ruff now reports syntax errors for emoji identifiers in your code, the same as CPython.
+
+### Improved GitLab fingerprints ([#7203](https://github.com/astral-sh/ruff/pull/7203))
+
+GitLab uses fingerprints to identify new, existing, or fixed violations. Previously, Ruff included the violation's position in the fingerprint. Using the location has the downside that changing any code before the violation causes the fingerprint to change, resulting in GitLab reporting one fixed and one new violation even though it is a pre-existing violation.
+
+Ruff now uses a more stable location-agnostic fingerprint to minimize that existing violations incorrectly get marked as fixed and re-reported as new violations.
+
+Expect that GitLab reports each pre-existing violation in your project as fixed and a new violation in your Ruff upgrade PR.
```
Comment (zanieb) [inline BREAKING_CHANGES.md:16]:
```suggestion
Expect GitLab to report each pre-existing violation in your project as fixed and a new violation in your Ruff upgrade PR.
```

**Review by zanieb — APPROVED** [review]

**Pull request merged**